conu/backend/docker/container.py

```diff
@@ -475,3 +490,69 @@ def write_to_stdin(self, message):
             self.popen_instance.stdin.flush()
         except subprocess.CalledProcessError as e:
             raise ConuException(e)
+
+    def get_container_metadata(self):
```

Comment:
I believe we discussed this with @dhodovsk and my suggestion was to break backwards compatibility: .get_metadata method would return instance of the Metadata class, and .inspect method would yield the dict from docker. WDYT?